GltfExport.cs

```diff
@@ -64,93 +68,103 @@ protected override void SolveInstance(IGH_DataAccess DA)
                 Message = "Scene";
             }
 
-            FileGltfWriteOptions optionsGltf =
+            FileGltfWriteOptions? optionsGltf =
                 options != null
-                && options! is FileGltfWriteOptions ?
+                && options is FileGltfWriteOptions ?
                 options as FileGltfWriteOptions : DefaultOptions();
 
-            if (!export) return;
-            string res = "";
+            if (!export || _executing)
+            {
+                if(_executing) _lastResult = "executing in background...";
+                goto Outputs;
+            }
+            _executing = true;
             List<Guid> ids = [];
             RhinoDoc doc = RhinoDoc.ActiveDoc;
-            try
+            Task.Run(() =>
```

Comment:
# 👍 Threading - Putting the try-catch block inside Task.Run()
 - good pattern to isolate error handling within the asynchronous operation itself
    - main thread stays unblocked by exceptions thrown during the execution of the task
    - allows to continue processing other operations